ui/src/components/home/HomeContainer.tsx

```diff
@@ -1,89 +1,156 @@
-              description="Find resources for worship here"
-              Icon={TextSnippetIcon}
-              route="/resource"
-            />
+  return (
+    <>
+      <Box
+        sx={{
+          width: '100%',
+          flexGrow: 1,
+          display: 'flex',
+          justifyContent: 'center',
+          alignItems: 'flex-start',
+          margin: 'auto',
+          paddingBottom: '1em',
+          paddingTop: ['0em', '2em'],
+          px: ['0em', '1em'],
+        }}
+      >
+        <Stack direction={'column'}>
+          <Stack direction={['column', 'row']} spacing={2} gap={['25px', '12px']}>
+            <Box
+              sx={{
+                pt: '2.5em',
+                pb: ['0.5em', '2.5em'],
+                px: '2em',
+                borderRadius: ['0px', '30px'],
+                background: [
+                  'linear-gradient(180deg, rgba(51, 45, 71, 0.00) 0%, #171717 88.5%), linear-gradient(158deg, rgba(0, 0, 0, 0.00) 31.44%, rgba(148, 111, 255, 0.20) 80.34%), radial-gradient(111.68% 110.13% at 66.1% 8.28%, rgba(154, 118, 255, 0.20) 36.5%, rgba(0, 0, 0, 0.20) 64%), #1F1F1F',
+                  'linear-gradient(158deg, rgba(0, 0, 0, 0.00) 31.44%, rgba(148, 111, 255, 0.20) 80.34%), radial-gradient(111.68% 110.13% at 66.1% 8.28%, rgba(154, 118, 255, 0.20) 36.5%, rgba(0, 0, 0, 0.20) 64%), #1F1F1F',
+                ],
+                width: ['100%', '50%'],
+              }}
+              position="relative"
+            >
+              <Typography variant="h1" sx={{ pb: 2, fontSize: ['24px', '40px'] }}>
+                Welcome to {isDesktop && <br />}
+                HMCC Worship
+              </Typography>
+              <Typography
+                sx={{ fontSize: ['14px', '16px'], fontFamily: 'DM Sans, sans-serif' }}
+                marginBottom={['2em', '0em']}
+                fontWeight={400}
+              >
+                Here is where you can find lyrics and chords for worship music! Go on and worship
+                God!
+              </Typography>
+              <Typography
+                position={['static', 'absolute']}
+                sx={{ fontSize: ['12px', '16px'], fontFamily: 'DM Sans, sans-serif' }}
+                fontWeight={400}
+                bottom={35}
+              >
+                Harvest Mission Community Church
+              </Typography>
+            </Box>
+            <Box
+              sx={{
+                marginBottom: '20px',
+                width: '60%',
+                display: ['flex', 'none'],
+                alignItems: 'center',
+                alignSelf: 'center',
+                border: '1px solid #D0BCFE',
+                backgroundColor: '#4A4458',
+                borderRadius: '28px',
+                px: 1,
+              }}
+              onClick={onSearchOpen}
+            >
+              <IconButton sx={{ color: '#b39ddb' }}>
+                <SearchIcon />
+              </IconButton>
+              <Box
```

Comment:
![image](https://github.com/user-attachments/assets/5404c312-63fd-47ab-8363-711a3940bb87)

the mobile nav bar seems to be a bit cut